spec/rubocop/cop/style/yoda_condition_spec.rb

```diff
@@ -324,6 +324,10 @@
       expect_no_offenses('bar === "foo"')
     end
 
+    it 'accepts equality check method is used without RHSl' do
```

Comment:
RHSl? Perhaps `argument` is the better term here.

Reply:
Sure! I've update this PR.